### PR TITLE
add navbar to layout

### PR DESCRIPTION
**Pull request by miguelrodrigues22**
Added the Navbar according with the design provided.

### CODE REVIEW

File: components/Layout.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 import Header from 'components/Header'
+import Navbar from 'components/Navbar'
 import PropTypes from 'prop-types'
 import styles from '../styles/Home.module.css'
 
 const Layout = ({ children }) => {
   return (
     <div className={styles.container}>
-      <Header />
+      {/* <Header /> */}
```
Comment (hugo-serra) [inline components/Layout.js:10]: whis is this removed?

File: components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+  { name: 'Your Profile', href: '#' },
+  { name: 'Settings', href: '#' },
+  { name: 'Sign out', href: '#' },
+]
+
+function classNames(...classes) {
+  return classes.filter(Boolean).join(' ')
+}
+
+export default function Navbar() {
```
Comment (hugo-serra) [inline components/Navbar.js:26]: Let's split this into smaller components

**Review by hugo-serra — APPROVED** [review]

File: components/Layout.js (hunk 1 above)
Comment (miguelrodrigues22) [reply components/Layout.js:10]: added the navbar component instead

File: components/Navbar.js (hunk 2 above)
Comment (miguelrodrigues22) [reply components/Navbar.js:26]: is it worth it for a function so small?

**Pull request closed without merging**